src/pages/RFOX/components/AddressSelection.tsx

```diff
@@ -197,16 +202,28 @@ export const AddressSelection: FC<AddressSelectionProps> = ({
         isDisabled={!maybeSelectedRuneAddress}
         value={maybeSelectedRuneAddress ?? ''}
       >
-        <AccountDropdown
-          defaultAccountId={maybeRuneAccountId}
-          assetId={thorchainAssetId}
-          onChange={handleAccountIdChange}
-          boxProps={boxProps}
-          buttonProps={buttonProps}
-        />
+        {accountNumber !== undefined ? (
+          <AccountDropdown
+            defaultAccountId={maybeRuneAccountId}
+            assetId={thorchainAssetId}
+            onChange={handleAccountIdChange}
+            boxProps={boxProps}
+            buttonProps={buttonProps}
+          />
+        ) : maybeSelectedRuneAddress ? (
+          <Tag colorScheme='gray'>
+            <MiddleEllipsis value={maybeSelectedRuneAddress} />
+          </Tag>
+        ) : null}
```

Comment:
ocd: Can we avoid nested ternary 🙏🏽

Reply:
Done in [73986b8](https://github.com/shapeshift/web/pull/7734/commits/73986b85950ba203adfc4172bc2a6ca5c130a29b)